<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelect</class>
 <widget class="QDialog" name="DlgSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抽奖</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="m_ctlDisplay">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>18600000000</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnStart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
     <property name="icon">
      <iconset resource="LotterySimulation.qrc">
       <normaloff>:/Resouces/Resources/play.png</normaloff>:/Resouces/Resources/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_lstResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LotterySimulation.qrc"/>
 </resources>
 <connections/>
</ui>
